cmd/proximo-server/config.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"github.com/uw-labs/proximo"
+	"github.com/uw-labs/proximo/proto"
+	"github.com/uw-labs/substrate"
+	"golang.org/x/crypto/bcrypt"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	ErrUnauthorized = errors.New("unauthorized")
```

Comment:
It would be nicer to make this a proper gRPC error, ie something like 
```suggestion
	ErrUnauthorized = status.Errorf(codes.PermissionDenied, "unauthorised")
```